src/utils/context.rs

```diff
@@ -9,12 +8,12 @@ pub struct Context<'r> {
     uri: String,
     host: Option<String>,
     cache: ServiceCache<'r>,
-    client: State<'r, reqwest::Client>,
+    client: &'r reqwest::Client,
```

Comment:
Why don't we wrap this in `State` anymore?

Reply:
State doesn'T exist anymore